app/src/components/market/sections/market_sell/market_sell.tsx

```diff
@@ -74,6 +74,13 @@ const MarketSellWrapper: React.FC<Props> = (props: Props) => {
     setIsNegativeAmountShares(formatBigNumber(amountShares || Zero, collateral.decimals).includes('-'))
   }, [amountShares, collateral.decimals])
 
+  useEffect(() => {
+    setOutcomeIndex(defaultOutcomeIndex)
+    setAmountShares(null)
+    setAmountSharesToDisplay('')
+    // eslint-disable-next-line
```

Comment:
We also need to update `selectedOutcomeBalance` so that 'Your Shares' is updated